tools/cloud_functions/bq_table_snapshots/terraform/terraform.tfvars

```diff
@@ -1,6 +1,6 @@
-project_id                = "YOUR_CLOUD_FUNCTIONS_PROJECT_ID"
-storage_project_id        = "YOUR_BIGQUERY_PROJECT_ID"
-source_dataset_name       = "YOUR_BIGQUERY_DATASET_ID"
-target_dataset_name       = "YOUR_SNAPSHOT_DATASET_ID"
-crontab_format            = "10 * * * *"
-seconds_before_expiration = 604800
+project_id                = "yardlink-data-prod"
+storage_project_id        = "yardlink-data-prod"
+source_dataset_name       = "oms_raw_public"
+target_dataset_name       = "oms_raw_public_history"
+crontab_format            = "0 0 * * 0"
+seconds_before_expiration = 5443200
```

Comment:
should this be 7776000 so expiry of 90 days? like the default above?

Reply:
I think i set it to be 9 weeks rather than the full 90 days which should mean we only have 1 extra week at any one time....